### PR TITLE
minor changes with document uploader to database

### PR DESCRIPTION
**Pull request by jakharkaran**

### CODE REVIEW

File: imports/route/route.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
 Router.route('/', function () {
-  this.render('home');
+	this.render('home');
```
Comment (hatimak) [inline imports/route/route.js:2]: Are you sure there is a `;` at the end of statement? IMO, it should be a `,`. Look for other similar lines below as well.

File: imports/ui/body.js [hunk 2]
```diff
@@ -3,11 +3,11 @@ import './body.html'
 import { Template } from 'meteor/templating';
 import { Vendors } from '../api/userdata.js';
 
-Template.body.onCreated(function bodyOnCreated() {
+Template.register.onCreated(function bodyOnCreated() {
     console.log('Body created')
```
Comment (hatimak) [inline imports/ui/body.js]: `'Register created'`?

File: imports/route/route.js (hunk 1 above)
Comment (jakharkaran) [inline imports/route/route.js:2]: yep.
reference(quick start section)
: https://github.com/iron-meteor/iron-router

File: imports/ui/body.js (hunk 2 above)
Comment (jakharkaran) [inline imports/ui/body.js]: yes u can change it to that. no template 'body' exists now. the content was included in other template 'register' in the last pull request. hence 'body' replace by 'register'.

**Pull request merged**